teamworkApp/app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>TeamworkApp</title>
-    <%= csrf_meta_tags %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
-
-  <body>
-    <%= yield %>
-  </body>
-</html>
+<head>
+  <title>Teamwork Analysis</title>
+  <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track" => true %>
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+	  <h1> Hello, welcome to our teamwork site :) </h1>
+	  <style>body{padding-top:70px;}</style>
+	</body>
+	</html>
```

Comment:
Do you maybe want to add a picture of the site as it currently appears to the review? that way we can see exactly what changes are occurring? Otherwise, it looks good.